api/config/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,6 +1,13 @@
 //go:build !ignore_autogenerated
 // +build !ignore_autogenerated
 
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+
+
 // Code generated by controller-gen. DO NOT EDIT.
```

Comment:
Surely the linter is complaining about all these extra newlines? (Because `gofmt` would remove them)

Reply:
This was an artifact of how controller-gen was working, fixed.